docker/base.Dockerfile

```diff
@@ -122,8 +122,8 @@ ENV FSLDIR=/usr/share/fsl/5.0 \
 
 # Installing and setting up ANTs
 RUN mkdir -p /opt/ants && \
-    curl -sSL "https://github.com/stnava/ANTs/releases/download/v2.1.0/Linux_Ubuntu14.04.tar.bz2" \
-    | tar -xjC /opt/ants --strip-components 1
+    curl -sSL "https://www.dropbox.com/s/2f4sui1z6lcgyek/ANTs-Linux-centos5_x86_64-v2.2.0-0740f91.tar.gz?dl=1" \
+    | tar -xzC /opt/ants --strip-components 1
```

Comment:
check the command i added in the other PR: this may install ants in `/opt/ants/ants`